multimedia/csfml/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                csfml
+version             2.5.1
+revision            0
+categories          multimedia devel
+maintainers         openmaintainer
```

Comment:
As per the lint warnings, you cannot have a `maintainers` block with only `openmaintainer`.  If you are not adding yourself as maintainer, then the block must be set as `nomaintainer`.